clients/runner/src/runner.rs

```diff
@@ -309,7 +309,7 @@ impl Runner {
 			.storage()
 			.at_latest()
 			.await?
-			.fetch_raw(&lookup_bytes)
+			.fetch_raw(&*lookup_bytes)
```

Comment:
Should we just clone then?
```suggestion
			.fetch_raw(lookup_bytes.clone())
```